openshift-ci/run_e2e.sh

```diff
@@ -5,11 +5,16 @@ source ${ingress_node_firewall_dir}/common.sh
 
 TESTS_REPORTS_PATH=${TESTS_REPORTS_PATH:-/logs/artifacts/}
 export OO_INSTALL_NAMESPACE=${OO_INSTALL_NAMESPACE:-"openshift-ingress-node-firewall"}
-export IS_OPENSHIFT="true"
 
 worker_node=$(oc get node | grep worker | head -n 1 | awk '{print $1}')
 ovnkube_node_pod=$(oc get pod -n openshift-ovn-kubernetes -owide | grep ovnkube-node | grep "${worker_node}" | awk '{print $1}')
-export NODE_INTERFACE=$(oc exec -n openshift-ovn-kubernetes "${ovnkube_node_pod}" ovs-vsctl list-ports br-ex | grep -v br-ex)
+# E2E tests test allowing / denying Pod IPs. This is not the typical use case for INF but it gives us flexibility and
+# control over the test cases that we would not enjoy if we attached to any other interface.
+# Pod IPs are visibly by the XDP program after they have been de-encapsulated and for OVN-Kubernetes as the CNI - this is a GENEVE interface.
+# Since OVN-Kubernetes is the default CNI for OCP, we set the interface to test against as the GENEVE interface.
+export NODE_INTERFACE=genev_sys_6081
```

Comment:
because we know in advance what the interface name is now so theres no need to look it up

Reply:
While this might be ok for ci but it’s not how will users use it can we at least comment out the old way till he have rhel9 then try it again on +ve side having ci passing is good